packages/console/src/command-history/CommandHistory.test.jsx

```diff
@@ -52,20 +52,23 @@ function mountItems(itemLength = 10) {
   return wrapper;
 }
 
+function getCommandItem(index) {
+  return `Command ${index}`;
+}
```

Comment:
Could have gone one step further here, and had `getCommandItem` return the screen.getByText, e.g.:

```suggestion
function getCommandTextindex) {
  return `Command ${index}`;
}

function getCommandItem(index) {
  return screen.getByText(getCommandText(index));
}
```